src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFindReplaceWidget.css

```diff
@@ -139,7 +139,8 @@
 	background-color: inherit !important;
 }
 
-.monaco-workbench .simple-fr-find-part-wrapper .find-filter-button {
+.monaco-workbench .simple-fr-find-part-wrapper .find-filter-button,
+.monaco-workbench .search-container .find-filter-button {
```

Comment:
changed the CSS here to also support the find widget when it's in the search viewlet.

Reply:
Since we are moving Filter view out to search, I think the CSS for search container should also be moved out.